### ISSUE TITLE
Add extra outputs to mrc_visual_min_max

### ISSUE THREAD

**Issue by philipmac** [niaid/tomojs-pytools#24]
Add: Floor and Limit

Also, prefix all keys with string "neuroglancerPrecomputed"

Leaving: 

```
    neuroglancerPrecomputedFloor - the physical minimum pixel value in the pyramid - (int)
    neuroglancerPrecomputedLimit - the physical maximum pixel value in the pyramid - (int)
    neuroglancerPrecomputedMin - the default lower limit for the Neuroglancer intensity scale - (int)
    neuroglancerPrecomputedMax - the default upper limit for the Neuroglancer intensity scale - (int)

```

**Comment by blowekamp** [comment]
@philipmac  Please review PR #26 to see if it meets you requirements.

**Comment by blowekamp** [comment]
closed by #24
